src/sentry/rules/actions/utils.py

```diff
@@ -42,6 +46,22 @@ def generate_diff_labels(
     return changed_data
 
 
+def get_frequency_label(value_str: str | None) -> str | None:
```

Comment:
Whatcha think about moving this to [utils/dates.py](https://github.com/getsentry/sentry/blob/master/src/sentry/utils/dates.py)? then we could name it like `format_minutes_to_str` or something.

Reply:
I was thinking about that, but since I'm not handling anything outside of our hardcoded values for issue alert rules I think it should at least stay in that area.